### ISSUE TITLE
Update maintenance versions

### ISSUE THREAD

**Issue by mdelapenya** [elastic/e2e-testing#749]
- [ ] 7.x --> 7.x-SNAPSHOT
- [ ] 7.11.x --> 7.11-SNAPSHOT
- [ ] 7.10.x --> 7.10-SNAPSHOT

@EricDavisX @michalpristas is this correct?

**Comment by EricDavisX** [comment]
yes, looks right to me.  In a few days during this part of the dev cycle we'll branch for 7.12 and we'll have a:
7.12.x --> 7.12-SNAPSHOT
reference as well.

and of course:
8.x --> 8.0-SNAPSHOT

**Comment by mdelapenya** [comment]
Closing, as the versions where added
